ivy/functional/backends/numpy/elementwise.py

```diff
@@ -12,8 +12,14 @@
 
 @_scalar_output_to_0d_array
 def abs(
-    x: Union[float, np.ndarray], /, *, out: Optional[np.ndarray] = None
+    x: Union[float, np.ndarray],
+    /,
+    *,
+    out: Optional[np.ndarray] = None,
+    where: Optional[Union[bool, np.ndarray]] = True,
 ) -> np.ndarray:
+    if where:
```

Comment:
I think we can just make one function call to `np.absolute`, given that it supports the `out` argument natively